### PR TITLE
Issue 1601 - Update runs tab to match new api schemas

### PR DESCRIPTION
**Pull request by jennifer-shehane**
- added 'object-schema' project, began updating fixture to match schema
- add timer display component
- add some util functions
- update tests to actual test the run data
- capital cased some messaging in the runs

### CODE REVIEW

File: packages/desktop-gui/src/lib/utils.js [hunk 1]
```diff
@@ -85,13 +88,31 @@ module.exports = {
...
-    let durationHours = duration.hours() ? `${duration.hours()}h ` : ''
-    let durationMinutes = duration.minutes() ? `${duration.minutes()}m ` : ''
-    let durationSeconds = duration.seconds() ? `${duration.seconds()}s ` : ''
+    const durationSecs = duration.seconds() ? `${duration.seconds()}` : ''
+    const durationMins = duration.minutes() ? `${duration.minutes()}` : ''
+    const durationHrs = duration.hours() ? `${duration.hours()}` : ''
+
+    const total = _.compact([
+      durationHrs,
+      !!durationHrs || padMinutes ? padStart(durationMins, 2, '0') : durationMins,
```
Comment (bahmutov) [inline packages/desktop-gui/src/lib/utils.js]: oh my, this expression is complicated

Comment (jennifer-shehane) [reply packages/desktop-gui/src/lib/utils.js]: It is! I mean, all I want is this logic in this spec to pass, so if there's a less complicated way, I'm game -> https://github.com/cypress-io/cypress/pull/1604/files#diff-6a3f8282b23f89f40bb2248f3e7d5c64

**Comment by jennifer-shehane** [comment]
@bahmutov This last test is failing because the CI env's timezone is not set to `America/New_York`. I'm not as familiar with this project's `circle.yml` file. Could you update the desktop tests to use this timezone?

**Comment by bahmutov** [comment]
Will do, I did it somewhere some time ago, if only I could remember where ...

**Comment by bahmutov** [comment]
hahaha, of course, I set it in the dashboard repo
```
$ grep TZ circle.yml -B 5
    docker:
      - image: cypress/base:6
        environment:
          TERM: xterm
          npm_config_loglevel: warn
          TZ: "/usr/share/zoneinfo/America/New_York"
```

**Pull request merged**